packages/gatsby-plugin-graphql-config/.babelrc

```
@@ -0,0 +1,3 @@
+{
+  "presets": [["babel-preset-gatsby-package", { "browser": true }]]
```

`browser` option is for situations when the build target is browser not node, so unnecessary.

```suggestion
  "presets": [["babel-preset-gatsby-package"]]
```